packages/neutron-sdk/src/bindings/dex/types.rs

```diff
@@ -116,7 +116,9 @@ pub struct MultiHopRoute {
 #[serde(rename_all = "snake_case")]
 pub struct Params {
     pub fee_tiers: Vec<u64>,
-    pub max_true_taker_spread: Option<PrecDec>,
+    pub paused: Option<bool>,
```

Comment:
Maybe we can remove `omitempty` tag on the Golang side?
I don't see any reason why this field should be an `Option<>`